server/venueless/api/views.py

```diff
@@ -73,3 +74,30 @@ def patch(self, request, **kwargs):
             lambda: async_to_sync(notify_world_change)(request.world.id)
         )
         return Response(serializer.data)
+
+    def post(self, request, **kwargs):
+        """POST on a detail view is, for now, inteded to execute actions, similar
+        to how you would do it with @action on a real ViewSet.
+
+        The request is expected to pass an 'action' attribute, and optionally content
+        in a 'data' attribute. Supported actions:
+            - schedule_update
```

Comment:
Any reason for not just creating another `APIView` with its own URL? Would feel much cleaner.